src/main/resources/languages/french.xml

```diff
@@ -296,7 +296,7 @@
     </string>
     <string>
         <key>SierpinskiTriangleName</key>
-        <value>Fractale du triangle de Sierpinski</value>
+        <value>Triangle de Sierpiński</value>
         <hint>[name of generator]</hint>
         <!--	"SierpinskiTriangleName"
```

Comment:
what is the purpose of those comments ? it will be outdated very shortly when the code is updating.
As the `<key>` is almost unique in the code, it's trivial to find where it is used by doing a search, isn't it ?

Reply:
I haven't deleted them for the days when I would like to work on renaming key to correspond to https://github.com/MarginallyClever/Makelangelo-software/discussions/392#discussion-3761569 
>  if a translated string is only used in one class it should be ClassName.KeyName

... to have an idea of the extent of the work to be done 
... as normally I have something to regenerate them automatically, I will remove them.